e2e/replay/msg_persister.go

```diff
@@ -44,25 +44,25 @@ func (r *replayMessagePersister) saveMetaData(nodeNames *[]string) error {
 	return err
 }
 
-// saveCachedMessages saves currently cached messages and timeouts to .flow file
-func (r *replayMessagePersister) saveCachedMessages() error {
+// SaveCachedMessages saves currently cached messages and timeouts to .flow file
+func (r *MessagePersister) SaveCachedMessages() error {
```

Comment:
Seems like this doesn't need to be public eventually (it is called by `MessageNotifier`, which is in the `replay` package as well)

Reply:
Done